Makefile

```diff
@@ -0,0 +1,19 @@
+SHELL := /bin/bash
+include version.mk
+
+IMAGE_URI=quay.io/redhat/managed-prometheus-exporter-initcontainer
```

Comment:
@jewzaam If we moved all variables to a separate make file which can be unique for each repo, we will be able to use a generic makefile for builds across all of our repos

Reply:
I expect makefiles to vary between repositories, there are going to be things that are specific to each codebase that has to be done.  The common bits can be included as was done here.  They're focused bits of common functionality and if you need it you pull it from the shared "library" (whatever that is eventually).  This initcontainer, metrics exporters, and operators all have different needs.

Reply:
It's common to actually separate out the image name from the repo part of the URI:
```
IMAGE_REPO=quay.io/redhat
IMAGE_NAME=managed-prometheus-exporter-initcontainer
```

This allows you to override it at runtime to push to, for example, a local or test store:
```
make build IMAGE_REPO=quay.io/cblecker
```